### DISCUSSION TITLE
Cobertura de tests

### DISCUSSION THREAD

**Discussion by MarioArocaPaez** [perealberto/decide-single-zambrano#83] [General]
Chicos os dejo aquí el test de cobertura de lo que llevamos ahora mismo. Tenemos un 80% no está mal. Lo dejo aquí para que nos vayamos organizando un poco como seguir haciendo tests
![imagen](https://github.com/3ralon/decide-single-zambrano/assets/73229438/c8ac52c3-d6c6-46cf-ad06-89a1bee6c4ed)
![imagen](https://github.com/3ralon/decide-single-zambrano/assets/73229438/a0858de1-d0b4-4e79-bcf6-abe3a6e72d82)
![imagen](https://github.com/3ralon/decide-single-zambrano/assets/73229438/97234545-09ac-4880-aca4-dd47057055df)
![imagen](https://github.com/3ralon/decide-single-zambrano/assets/73229438/8a342818-751d-48ad-87e3-ed94464082cc)

**Comment by laurolmer** [comment]
Veo que tenemos bastante cobertura, más que añadir tests nuevos mi propuesta es de hacer que las pruebas que tenemos ahora mismo tengan mayor variedad y prueben más casos, ya que hay tests que son un poco "happy path"

**Comment by perealberto** [comment]
Hace poco volví a lanzar un **coverage** y anoté los módulos con menos porcentaje de cobertura. Lo dejo por aquí para que tengamos constancia de ello.

![image](https://github.com/3ralon/decide-single-zambrano/assets/128732971/dccfc41f-0bd3-497e-b459-c16a70ff80d1)
